packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/NftFilter/index.tsx

```diff
@@ -162,33 +163,31 @@ const NftFilter: React.FC<Props> = ({
         </FilterHeaderText>
         <IconWrapper isFilterOpen={isFilterOpen}>
           {isFilterOpen ? (
-            <Icon label='filter-control' color='grey500'>
-              <IconCloseCircleV2
-                role='button'
-                onClick={() => {
-                  setIsFilterOpen(false)
-                  analyticsActions.trackEvent({
-                    key: Analytics.NFT_LEFT_MENU_CLOSED,
-                    properties: {}
-                  })
-                }}
-                cursor='pointer'
-              />
-            </Icon>
+            <IconCloseCircleV2
+              label='filter-control'
+              color={PaletteColors['grey-500']}
+              role='button'
+              onClick={() => {
+                setIsFilterOpen(false)
+                analyticsActions.trackEvent({
+                  key: Analytics.NFT_LEFT_MENU_CLOSED,
+                  properties: {}
+                })
+              }}
+            />
           ) : (
-            <Icon label='filter-control' color='grey500'>
-              <IconFilter
-                role='button'
-                onClick={() => {
-                  setIsFilterOpen(true)
-                  analyticsActions.trackEvent({
-                    key: Analytics.NFT_LEFT_MENU_EXPANDED,
-                    properties: {}
-                  })
-                }}
-                cursor='pointer'
-              />
-            </Icon>
+            <IconFilter
+              label='filter-control'
+              color={PaletteColors['grey-500']}
+              role='button'
+              onClick={() => {
+                setIsFilterOpen(true)
+                analyticsActions.trackEvent({
+                  key: Analytics.NFT_LEFT_MENU_EXPANDED,
+                  properties: {}
+                })
+              }}
+            />
```

Comment:
This is another ternary repetition case. Hidden inside, a function that can be taken outside the render method and handled either by checking there the `isFilterOpen`,  or creating two similar functions, `handleExpandMenu` and `handleCloseMenu`